### PR TITLE
[Android/Single] Do not turn off a large code block with #if

### PR DESCRIPTION
**Pull request by myungjoo**
Separate the file and build independently instead.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

CC: @jaeyun-jung : This requires your review.

### CODE REVIEW

File: api/android/api/src/main/jni/Android.mk [hunk 1]
```diff
@@ -43,6 +41,8 @@ ifeq ($(NNSTREAMER_API_OPTION),single)
 NNS_API_FLAGS += -DNNS_SINGLE_ONLY=1
 endif
 
+include $(NNSTREAMER_ROOT)/jni/nnstreamer.mk
```
Comment (jaeyun-jung) [inline api/android/api/src/main/jni/Android.mk:44]: plz move line 37 under this line (NNSTREAMER_VERSION is defined in nnstreamer.mk)
NNS_API_FLAGS += -DVERSION=\"$(NNSTREAMER_VERSION)\"
NNSTREAMER_VERSION

File: jni/nnstreamer.mk [hunk 2]
```diff
@@ -25,6 +25,10 @@ NNSTREAMER_COMMON_SRCS := \
     $(NNSTREAMER_GST_HOME)/nnstreamer_subplugin.c \
     $(NNSTREAMER_GST_HOME)/tensor_common.c
 
+ifneq ($(NNSTREAMER_API_OPTION),single)
```
Comment (jaeyun-jung) [inline jni/nnstreamer.mk:28]: how about adding 'tensor_common_pipeline.c' in NNSTREAMER_PLUGINS_SRCS?

File: api/android/api/src/main/jni/Android.mk (hunk 1 above)
Comment (myungjoo) [reply api/android/api/src/main/jni/Android.mk:44]: Applied. Thanks!

File: jni/nnstreamer.mk (hunk 2 above)
Comment (myungjoo) [reply jni/nnstreamer.mk]: Moved to NNSTREAMER_PLUGINS_SRCS along with a few comments to avoid confusion.

**Pull request merged**